src/pages/index.tsx

```diff
@@ -43,6 +44,8 @@ export default Index;
 
 export const getServerSideProps: GetServerSideProps = withSSRLogger(
   async (context) => {
+    configureReq(context.req);
```

Comment:
Why is there no try/catch there?

Reply:
Because there is no way an error happens there, I think, as the function is only declaring an object, and passing it to the React component (_already regret those words_).

But you are right, calling `configureReq` has no effect there.